pkg/mount/README.md

```diff
@@ -0,0 +1,34 @@
+mount
+=====
+
+Mount block devices for use in Linuxkit
+
+**NOTE: At this time, block devices may only be mounted in `/var`**
```

Comment:
Yes it would be better to document this. We could also add some more standard mount points eg `/mnt`.